ui/src/app/shared/translate/en.ts

```diff
@@ -3,6 +3,7 @@ export const TRANSLATION = {
         Grid: "Grid",
         GridBuy: "Buy from grid",
         GridSell: "Sell to grid",
+        GridMode: "No Grid Connection!",

```

Comment:
The name should be something like "OffGrid" or "NoGridConnection" instead of "GridMode"